linux_os/guide/system/software/integrity/crypto/harden_ssh_client_crypto_policy/rule.yml

```diff
@@ -36,3 +37,16 @@ ocil: |-
     <pre>PubkeyAcceptedKeyTypes ssh-rsa,ecdsa-sha2-nistp384,ecdsa-sha2-nistp256</pre>
     <pre>MACs hmac-sha2-512,hmac-sha2-256</pre>
     <pre>KexAlgorithms ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group14-sha1</pre>
+
+fixtext: |-
+    Configure the SSH client to override the system cryptographic policy.
+
+    Create file "/etc/ssh/ssh_config.d/02-ospp.conf" with the following content:
+
+    Match final all
+    RekeyLimit 512M 1h
+    GSSAPIAuthentication no
+    Ciphers aes256-ctr,aes256-cbc,aes128-ctr,aes128-cbc
```

Comment:
I think we will need some more review on this. I don't we think can in good faith recommend AES128-CBC in 2022. We might have to talk to the crypto team to get more details.

Reply:
Yes, good catch!

FTR it comes from https://github.com/ComplianceAsCode/content/pull/4681 which has been done for Common Criteria.

Reply:
We should just remove this rule. Comment no longer revlant.